frontend/public/components/edit-yaml.jsx

```diff
@@ -86,8 +92,27 @@ export const EditYAML_ = connect(stateToProps)(
         this.setState({ error, success: null });
       }
 
+      loadCSVs() {
+        const { namespace } = this.props.obj.metadata;
+        if (!namespace) {
```

Comment:
Can you check if this is a creation editor as well? We should skip this if creating a new resource.